spring-cloud-stream-schema-server/src/main/java/org/springframework/cloud/stream/schema/server/config/SchemaServerConfiguration.java

```diff
@@ -34,6 +36,7 @@
 @Configuration
 @EnableJpaRepositories(basePackageClasses = SchemaRepository.class)
 @EnableConfigurationProperties(SchemaServerProperties.class)
+@EntityScan(basePackageClasses = Schema.class)
```

Comment:
This can be removed as the fix is already available here: https://github.com/spring-cloud/spring-cloud-stream/commit/6c9976b77f5e54771516b62ed971382c3ffe36e5